.github/workflows/publish.yaml

```diff
@@ -102,12 +105,25 @@ jobs:
         password: ${{ secrets.DOCKERHUB_PASSWORD }}
 
     - uses: ./.github/actions/build-push-action
+      id: build-push-action
       with:
         context: deploy
         image-name: index.docker.io/replicated/replicated-sdk:v${{needs.get-tags.outputs.tag}}
         git-tag: v${{needs.get-tags.outputs.tag}}
         registry-username: ${{ secrets.DOCKERHUB_USER }}
         registry-password: ${{ secrets.DOCKERHUB_PASSWORD }}
+    
+    - name: 'Trim and Save Digest'
+    # build-push-action outputs the full image name and digest, but we need to save just the sha256 part
+      id: trim-and-save-digest
+      uses: actions/github-script@v7
+      with:
+       script: |
+          const fullDigest = "${{ steps.build-push-action.outputs.digest }}";
+          const digest = fullDigest.split('@')[1];
+          console.log(fullDigest);
```

Comment:
If we need to log these digests, should we at least prefix them with something to describe what they are? i.e. "Full digest" / "Trimmed digest", or something like that.

Reply:
Thanks for catching those. I had them for debugging and forgot to remove them.